### PR TITLE
Add gstreamer1.0-drpai recipe

### PR DESCRIPTION
**Pull request by matinlotfali**

### CODE REVIEW

**Review by mistyron — APPROVED** [review]
LGTM

File: recipes-core/images/mistysom-image.bbappend [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CORE_IMAGE_BASE_INSTALL += "pv kernel-modules weston weston-init weston-examples gtk+3-demo clutter-1.0-examples"
+CORE_IMAGE_BASE_INSTALL += "pv htop kernel-modules weston weston-init weston-examples gtk+3-demo clutter-1.0-examples"
```
Comment (mistyron) [inline recipes-core/images/mistysom-image.bbappend:1]: Do we really need `htop`? Or is it "just" added junk we don't need? If there's no good reason to add it, I vote for removing it

**Review by mistyron — CHANGES_REQUESTED** [review]
Requesting feedback on comment

File: recipes-core/images/mistysom-image.bbappend (hunk 1 above)
Comment (matinlotfali) [reply recipes-core/images/mistysom-image.bbappend:1]: It makes my life easier. I am not comfortable working with `top`.

Comment (mistyron) [reply recipes-core/images/mistysom-image.bbappend:1]: You're twisting my arm! 
Since it's a relatively small utility (~346KB),  we can live with it.

**Review by mistyron — APPROVED** [review]
LGTM

**Pull request merged**